ballerina/natives.bal

```diff
@@ -123,6 +135,25 @@ public isolated function printWarn(string msg, error? 'error = (), *KeyValues ke
     }
 }
 
+# Set the log output to a file. Note that all the subsequent logs of the entire application will be written to this file.
+# ```ballerina
+# log:setOutputFile("./resources/myfile.log");
+# log:setOutputFile("./resources/myfile.log", log:OVERWRITE);
+# ```
+#
+# + path - The path of the file
+# + option - To indicate whether to overwrite or append the log output
+public isolated function setOutputFile(string path, FileWriteOption option = APPEND) {
+    lock {
+        outputFilePath = path;
+    }
+    if option == OVERWRITE {
+        io:Error? result = io:fileWriteString(path, "");
+        if result is error {
+            printError("failed to set log output file", 'error = result);            }
```

Comment:
is there any reason to print and ignore the error here? if not shall we return the error.

Reply:
Updated in 3038cc1